_pages/clients/more-speech.md

```diff
@@ -0,0 +1,28 @@
+---
+title: more-speech
+permalink: /more-speech/
+web: 
+github: unclebob/more-speech
+instances:
+tags:
+- social
+- client
+- desktop 
+license: 
+---
+
+# A Nostr Browser in Clojure
+
+![more-speech](/images/more-speech.png)
```

Comment:
That scares people :D

Don't bother adding images to this repo at this point.

Reply:
Noted. Deleting images folder - moving all new work to generalInfo